languages/csharp-dotnet-2.2/template/__ALGO__.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Algorithmia.Client" Version="(1.0,)" />

```

Comment:
Can't we make this `1.0.0` instead of `1.0`?

Reply:
No, it would prevent us from making backwards compatible changes to the client and having it automatically propegate to existing algorithms. Same thing with the Algorithmia.Pipe.

Reply:
Actually I've updated this doc as the (1.0,) versioning doesn't actually do what I want it to do. From IRC the solution is `1.0.*` instead. No idea why that's not in the .NET docs anywhere...